src/styles/blocks/catalog.scss

```
@@ -0,0 +1,25 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+
+  &__cards {
+    display: flex;
+    flex-wrap: wrap;
+
+    max-width: 4 * $cardWidth + 3 * $cardMargin;
+  }
+
+  &__card {
+    margin-right: $cardMargin;
+    margin-bottom: 46px;
+
```

```suggestion
    height: fit-content;
```

Without setting proper height value your cards become larger than others if you have cards < 4 in last row.